Add manager challenge to interview assignments (#5)
hashicorp-interviews/education-assignments: manager/assignment.md

## Overview 

> This technical writing and editing project is part of your interview. It is designed to take between 1.5 - 2 hours.

Reviewer: I wonder if we should add a sentence to explain the purpose of this exercise？ Something like...

```suggestion
As a manager of product education engineering, you are accountable for ensuring content quality, which involves reviewing and providing clear feedback on your team's work. Therefore, this technical writing and editing project is part of your interview. It is designed to take between 1.5 - 2 hours.
```

Something to _hint_ that we're looking for the candidate's ability to coach the team. 

But if you think the recruiter will communicate that to candidates, please ignore this suggestion.

The instruction looks good!

## Instructions

> Imagine that a member of your team has published this [sample text](terraform-getting-started.md). Create a new repository containing the sample text, and open a PR to give your team member feedback, improve the content, correct the educational and technical errors, and implement the provided [writing style guide](../styling-guide-snippet.md) and [guide template](../guide-template.md). You may link to existing resources created by HashiCorp or the community, but do not plagiarize content from HashiCorp tutorials or documentation. 

Reviewer 2: ```suggestion
Imagine that a member of your team has published this [sample text](terraform-getting-started.md). You will open a PR to give your team member feedback, improve the content, correct the educational and technical errors, and implement the provided [writing style guide](../styling-guide-snippet.md) and [guide template](../guide-template.md). You may link to existing resources created by HashiCorp or the community, but do not plagiarize content from HashiCorp tutorials or documentation. 
```

Author: I'm leaving this one because we always get so many mis-submitted PRs. I think mentioning these mechanics early and often will help, especially since they're a weird workflow

Author: Fixing the run on sentence though! Hopefully this won't be as bad!

> **NOTE: Do not fork this repository.** Make a copy of all files, and create a fresh repository. This will prevent other candidates from easily finding your work.

Reviewer 2: ```suggestion
**NOTE: Do not branch or fork this repository.** Make a copy of all files, and create a fresh repository. This will prevent other candidates from easily finding your work.
```

To complete the assignment:

1. Create a new, public GitHub repository (do not fork this repository). Make sure the repository is publicly accessible so we can review your submission.
1. Before editing, commit [the initial version of the sample text](terraform-getting-started.md) to the main branch of the repository. 
1. Create a new branch and update the sample text to satisfy the style guide.
1. Create a pull request for your proposed changes to the text. Do not merge the pull request.

> 

Reviewer 2: ```suggestion
1. Include additional feedback you have for the team member in the body of the pull request.

```

Author: I was on the fence about this, is it too much hand holding? Part of the test here is if the candidate understands how to use a PR to give feedback. I don't want to be tricky but also I'm concerned that if we put this in writing we're going to loose some signal about their version control knowledge?

Author: Clarified the instructions above and I think that should walk the line
